runtime/vm/jvminit.c

```diff
@@ -2846,6 +2852,19 @@ VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved)
 #endif /* J9VM_INTERP_ATOMIC_FREE_JNI_USES_FLUSH */
 			TRIGGER_J9HOOK_VM_ABOUT_TO_BOOTSTRAP(vm->hookInterface, vm->mainThread);
 			/* At this point, the decision about which interpreter to use has been made */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+			if (vm->checkpointState.isCheckPointAllowed) {
+				if (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm)) {
+#if defined(OMR_GC_COMPRESSED_POINTERS)
+					vm->bytecodeLoop = criuBytecodeLoopCompressed;
+#endif /* OMR_GC_COMPRESSED_POINTERS */
+				} else {
+#if defined(OMR_GC_FULL_POINTERS)
+					vm->bytecodeLoop = criuBytecodeLoopFull;
+#endif /* OMR_GC_FULL_POINTERS */
```

Comment:
These two `#endif` comments should be corrected.

Reply:
Updated